tests/e2e/pageobjects/dashboard/Dashboard.ts

```diff
@@ -58,11 +61,37 @@ export class Dashboard {
         await this.workspaces.waitWorkspaceListItemAbcence(workspaceName);
     }
 
+     async stopAndRemoveWorkspaceWithSelectingTab(workspaceName: string) {
+        Logger.debug(`Dashboard.stopAndRemoveWorkspaceByUI "${workspaceName}"`);
+        await this.openDashboardTab()
+        await this.stopWorkspaceByUI(workspaceName);
+        await this.workspaces.deleteWorkspaceByActionsButton(workspaceName);
+        await this.workspaces.waitWorkspaceListItemAbcence(workspaceName);
```

Comment:
Typo: `Absense` > `Absence`

Reply:
Sorry, cannot find a typo in this place. Could you clarify this?